app/twig.php

```diff
@@ -0,0 +1,9 @@
+<?php
+require 'init.php';
+
+$pageVariables = [
+    'title' => 'Twig Templates',
+];
+
+echo $twig->render('test.twig', $pageVariables);
```

Comment:
just `echo` the output of the `render` function - here, we tell it to render the `test.twig` template, using the `$pageVariables` array!